github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -874,4 +874,65 @@ postsubmits:
           resources:
             requests:
               memory: "200Mi"
+    - name: post-project-infra-kubevirt-releasegraph
+      always_run: false
+      annotations:
+        testgrid-create-test-group: "false"
+      cluster: kubevirt-prow-control-plane
+      decorate: true
+      extra_refs:
+      - org: kubevirt
+        repo: project-infra
+        base_ref: main
+        workdir: true
+      labels:
+        preset-github-credentials: "true"
+      run_if_changed: "robots/cmd/perf-report-creator/shape.yaml"
+      spec:
+        containers:
+        - image: quay.io/kubevirtci/pr-creator:v20250320-f6c439c
+          env:
+          - name: GIMME_GO_VERSION
+            value: "1.22.5"
+          - name: TARGET_GITHUB_REPO
+            value: kubevirt/ci-performance-benchmarks
+          - name: GITHUB_TOKEN_PATH
+            value: /etc/github/oauth
+          command:
+          - /usr/local/bin/runner.sh
+          - /bin/sh
+          - -c
+          args:
+          - |
+            go build -o /usr/local/bin/perf-report-creator ./robots/cmd/perf-report-creator/...
+
+            RELEASE_VERSION=$(grep "releaseVersion:" robots/cmd/perf-report-creator/shape.yaml | awk '{print $2}' | tr -d '"')
```

Comment:
would it be better to send the shape.yaml to the go tool and have it parse instead of using awk here?
```
perf-report-creator weekly-graph \
  --resource=vmi \
  --weekly-reports-dir=${OUTPUT_DIR}/weekly \
  --plotly-html=true \
  --is-during-release=true \
  --release-config=shape.yaml
```

Reply:
ohh I see further down, the tool is doing that, the script needs to be changed accordingly.